packages/create-rslib/fragments/base/react-ts/rslib.config.ts

```diff
@@ -0,0 +1,46 @@
+import { pluginReact } from '@rsbuild/plugin-react';
+import { pluginSass } from '@rsbuild/plugin-sass';
+import { type LibConfig, defineConfig } from '@rslib/core';
+
+const shared: LibConfig = {
+  bundle: false,
+  dts: {
+    bundle: false,
+  },
+};
+
+export default defineConfig({
+  source: {
+    entry: {
+      index: ['./src/**', '!./src/env.d.ts'],
+    },
+  },
+  lib: [
+    {
+      ...shared,
+      format: 'esm',
+      output: {
+        distPath: {
```

Comment:
The templated package of `node-dual` not set different output folder for esm/cjs, and it should be same with specification in package.json